cfgov/cfgov/settings/base.py

```diff
@@ -232,11 +232,12 @@
 
 # List of finder classes that know how to find static files in
 # various locations.
-STATICFILES_FINDERS = (
+STATICFILES_FINDERS = [
     'sheerlike.finders.SheerlikeStaticFinder',
     'django.contrib.staticfiles.finders.AppDirectoriesFinder',
     'django.contrib.staticfiles.finders.FileSystemFinder',
-)
+    'compressor.finders.CompressorFinder',
```

Comment:
I think maybe this was removed but the `()` => `[]` change made the diff show it as an addition. @chosak can you comment?

Reply:
@jimmynotjim thanks for noticing. This should have been removed in #4084. Does this PR rely on django-compressor in some way? Maybe this got added back in due to a merge/rebase glitch?

Reply:
@chosak Yeah, I think git interpreted it as an addition when I did the rebase to master due to the parens to bracket change. Out of curiosity, do you we even need the parens to bracket change? Can I revert this entire file?

Reply:
@jimmynotjim unfortunately not. We do need the parens to bracket change because [this addition](https://github.com/cfpb/cfgov-refresh/pull/3887/files#diff-0cf61f1e9bb958eff38a3fae3fd25a3fR41) to the test settings in this PR wants to add to this list, and it can't if it's a tuple (using `()`).

Reply:
Ah ok, thanks for the clarification. I'll remove L:239